src/BankingApp.Data/Migrations/AccountDb/20240207210930_FirstAccountMigration.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace BankingApp.Data.Migrations.AccountDb
+{
+    public partial class FirstAccountMigration : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "Statements",
```

Comment:
observed that your table name for account name is statements instead of accounts and the features are for statement.cs class not account.cs class. is that in order?